addons/l10n_no/models/account_journal.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    invoice_reference_model = fields.Selection(selection_add=[('norway', 'Norway')])
```

Comment:
I see for Belgium, it is shortened to 'be'.  Maybe we can do 'no' here too?

Reply:
Yes right :+1: